core/sr-primitives/src/lib.rs

```diff
@@ -170,7 +170,7 @@ pub type ConsensusEngineId = [u8; 4];
 /// Permill is parts-per-million (i.e. after multiplying by this, divide by 1000000).
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug, Ord, PartialOrd))]
 #[derive(Encode, Decode, Default, Copy, Clone, PartialEq, Eq)]
-pub struct Permill(u32);
+pub struct Permill(pub u32);
```

Comment:
Why do you make this public?

I would rather add a method to get the value. (Otherwise someone may screws up the internal state)

Reply:
I forgot to put a comment about the why of it myself: to use it as a const value.